src/storage.rs

```diff
@@ -121,6 +123,48 @@ impl CQLSession {
         self.0.query(CREATE_EDGE_TABLE_QUERY)?;
         Ok(())
     }
+
+    fn paged_query(
+        &self,
+        query: String,
```

Comment:
Why is this String. According to original type, it's a trait bound with `ToString`. Could this just be `&str`?

Reply:
Modified. Thank you for pointing this out.